src/pages/HomeAbout.js

```diff
@@ -1,125 +1,196 @@
-import { Component } from 'react';
-import { a, div, h, hr, h1, h3, p, span, img, iframe } from 'react-hyperscript-helpers';
-import Mailto from 'react-protected-mailto';
-import { Config } from '../libs/config';
+import {Component} from 'react';
+import {div, h1, h3, p, img} from 'react-hyperscript-helpers';
 
 class HomeAbout extends Component {
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      powerBiUrl: ''
-    };
-  }
+  render() {
 
-  async componentDidMount() {
-    const powerBiUrl = await Config.getPowerBiUrl();
-    this.setState({ powerBiUrl: powerBiUrl });
-  }
+    const imageWrapper = {
+      padding: '0 2rem',
+      margin: '3rem 5rem 1rem',
+      height: 'auto',
+      width: '100%',
+      float: 'none'
+    }
 
-  render() {
+    const aboutSectionWrapper = {
+      fontFamily: 'Montserrat',
+      margin: '5rem auto',
+      overflow: 'auto',
+      textAlign: 'center',
+      color: '#1F3B50',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center'
+    }
+
+    const aboutSectionTitle = {
+      fontWeight: '600',
+      color: '#2899BC',
+      padding: '0 5rem'
+    }
+
+    const aboutSectionBody = {
+      padding: '0 5rem',
+      margin: '3rem auto 1rem'
+    }
 
     return (
       div({className: 'row home'}, [
-        div(
-          {className: 'col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1 col-sm-12 col-xs-12'},
-          [
-            h1({className: 'home-title'}, ['Data Use Oversight System']),
-            hr({className: 'home-line'}),
-            div({className: 'home-sections home-sections-table'}, [
+        div({className: 'col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1 col-sm-12 col-xs-12'}, [
+          div({}, [
+            div({style: aboutSectionWrapper}, [
+              h1({style: {fontWeight: '600'}}, ['Data Use Oversight System'])
+            ]),
+            div({style: aboutSectionWrapper}, [
+              h3({style: aboutSectionTitle}, ['Current state of data access']),
```

Comment:
Question, more for @solideoglori ... should this be capitalized like so?
```suggestion
              h3({style: aboutSectionTitle}, ['Current State of Data Access']),
```